test/markup/javascript/keyword-as-part-of-method-name.expect.txt

```diff
@@ -0,0 +1,2 @@
+<span class="hljs-keyword">setTimeout</span>();
+<span class="hljs-keyword">setInterval</span>();
```

Comment:
Is this correct? it should be built-in now, yes? Are you running the tests still?